roles/ceph-osd/tasks/check_mandatory_vars.yml

```diff
@@ -55,6 +56,36 @@
     - not osd_auto_discovery
     - devices|length == 0
 
+- name: verify lvm_volumes have been provided
+  fail:
+    msg: "please provide lvm_volumes to your osd scenario"
+  when:
+    - osd_group_name is defined
+    - osd_group_name in group_names
+    - lvm_osds
+    - not osd_auto_discovery
+    - lvm_volumes|length == 0
+
+- name: verify osd_objectstore is 'filestore' when using lvm_osds
```

Comment:
I'll just have this task before `verify lvm_volumes have been provided`so we stop right away. There is no need to evaluate `verify lvm_volumes have been provided` to fail after because of a wrong object store.